docs/Repository/README.md

```diff
@@ -0,0 +1,46 @@
+## Garden Linux Repositories
+
+Garden Linux is a Debian-based Operating System. Therefore it uses apt.
+
+The location of sources are at `/etc/apt/sources.list` respectively in the folder `/etc/apt/sources.list.d/`
+
+#### Default Repo:
+	deb http://repo.gardenlinux.io/gardenlinux dev main
+
+#### Generally:
+
+	deb http://repo.gardenlinux.io/gardenlinux $GARDENLINUX_EPOCH main
+
+`$GARDENLINUX_EPOCH` is generally `dev`, but the Garden Linux Version number can be defined here. It is the number of days since first Garden Linux release.
+
+<details>
+  <summary>More Info</summary>
+  
+Try running [bin/garden-version](bin/garden-version), to get the $GARDENLINUX_EPOCH value:
```

Comment:
relative link, so points to `docs/Repository/bin/garden-version`. maybe change to:
```
[bin/garden-version](../../bin/garden-version)
```

Reply:
that was actually the first version i had - i was unsure if it works, so i chose the full path. i changed this back to your suggestion.